### PR TITLE
Better DDOC_BLANKLINE companion

### PR DESCRIPTION
**Pull request by andralex**
This works in tandem with https://github.com/D-Programming-Language/dmd/pull/5344. Together, they produce the same html output for the site and standard library.

I had to do a few tweaks to the source files.

### CODE REVIEW

File: macros.ddoc [hunk 1]
```diff
@@ -2,9 +2,11 @@ _=Fundamental macros that apply to all generated formats
 
 ARGS = $0
 COMMA = ,
-COMMENT = 
+COMMENT =
```
Comment (andralex) [inline macros.ddoc]: This is spurious (due to my editor setup), but spaces at the end of the line are suspicious anyway.

File: std.ddoc [hunk 2]
```diff
@@ -37,3 +37,6 @@ JOIN_DOT_TAIL=$(JOIN_DOT $+)
 JOIN_DOT=.$1$(JOIN_DOT $+)
 TAIL=$+
 _=
+
+DDOC_BLANKLINE = $(BR)$(BR)$(LF)
```
Comment (andralex) [inline std.ddoc]: The extra LF in there is for compatibility with the existing generated HTML. We may eliminate it later if we find it fit.

File: macros.ddoc (hunk 1 above)
Comment (JakobOvrum) [inline macros.ddoc]: [It's part of our .editorconfig](https://github.com/D-Programming-Language/dlang.org/blob/master/.editorconfig), but I think we can override it for specific files if we need to.

**Comment by andralex** [comment]
Let me retire this and redo it after the bug push.

**Comment by andralex** [comment]
Back in swing!

**Comment by andralex** [comment]
egh, lemme work off a different branch

**Pull request closed without merging**